_aggregations/bucket/auto-interval-date-histogram.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Auto-date histogram
+parent: Bucket aggregations
+grand_parent: Aggregations
+nav_order: 15
+---
+
+# Auto-date histogram
```

Comment:
@msfroh @bowenlan-amzn @sandeshkr419 Would one of you be available to technical review this PR? The doc team is closing content gaps for aggregations and need a technical review before we can published the doc. Please note that I used internal tools to assist in code example generation and Dev Tools to test those examples. Thank you.